compiler/rustc_expand/src/base.rs

```diff
@@ -1497,7 +1497,7 @@ pub(crate) fn stream_pretty_printing_compatibility_hack(
             let mut parser = Parser::new(psess, stream.clone(), None);
             // No need to collect tokens for this simple check.
             parser
-                .parse_item(ForceCollect::No)
+                .parse_item(ForceCollect::No, AllowConstBlockItems::FIXME)
```

Comment:
this method is just for a specific hack that attempts to reparse items with a comment on it that it is intended to be removed anyway. Not supporting const blocks here seems fine, let's leave it at no.